metrics/metrics_test.go

```diff
@@ -82,6 +73,7 @@ func TestMeasurement(t *testing.T) {
 	time.Sleep(5)
 	measureSince("TestMeasurement1", now)
 
+	time.Sleep(20 * time.Millisecond)
```

Comment:
I'd like to go on record saying that this is ugly as hell and I hate myself for doing it